fe/src/test/java/org/apache/doris/catalog/ColocateTableTest.java

```diff
@@ -477,62 +377,4 @@ public void testDistributionColumnsType() throws Exception {
 
         catalog.createTable(childStmt);
     }
-
-    @Test
-    public void testParentTableNotExist() throws Exception {
-        testParentTableNotExist("t8");
-    }
-
-    @Test
-    public void testParentTableNotExistCaseSensitive() throws Exception {
-        testParentTableNotExist(tableName1.toUpperCase());
-    }
-
-    private void testParentTableNotExist(String tableName) throws Exception {
-        Map<String, String> properties = new HashMap<String, String>();
-        properties.put(PropertyAnalyzer.PROPERTIES_COLOCATE_WITH, tableName);
-
-        int bucketNum = 1;
-        CreateTableStmt parentStmt = new CreateTableStmt(false, false, dbTableName1, columnDefs, "olap",
-                new KeysDesc(KeysType.AGG_KEYS, columnNames), null,
-                new HashDistributionDesc(bucketNum, Lists.newArrayList("key1")), properties, null);
-        parentStmt.analyze(analyzer);
-
-        expectedEx.expect(DdlException.class);
-        expectedEx.expectMessage(String.format("Colocate table '%s' no exist", tableName));
-
-        catalog.createTable(parentStmt);
-    }
-
-    @Test
-    public void testParentTableType() throws Exception {
```

Comment:
Why remove `testParentTableType` method?  I think we still only support OlapTable.

Reply:
Oh, I though it was some 'parent-child-table' test. I will add it back

Reply:
OK